client/src/App.css

```diff
@@ -50,3 +50,58 @@
 .white {
   background: #FFFFFF;
 }
+
+.datetimepicker-dummy .datetimepicker-dummy-wrapper {
+  border: #FFFFFF;
+  background: #FFFFFF;
+  border-radius: 5px;
+}
+
+.datetimepicker-dummy.is-primary::before, .datetimepicker-dummy.is-primary:before {
+  background-color: #557769 !important;
```

Comment:
I know we talked about this, unfortunately I'm not experiencing enough in Frontend dev to know the best practice here, but I have a feeling using !important is not best practice. Okay with it for this project but if you are working in industry one day it would be best to learn the correct approach.